src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.jsx

```diff
@@ -638,6 +681,7 @@ const ImageClassificationObservationTable = ({
           onAnnotationSaveSuccess={fetchImages}
         />
       ) : undefined}
+      {removePhotoModal}
```

Comment:
Please set this to only render conditionally
```suggestion
      {isRemovePhotoModalOpen && removePhotoModal}
```

Reply:
the check has been set in a generic modal component
<img width="807" alt="Screenshot 2025-05-23 at 09 30 36" src="https://github.com/user-attachments/assets/c27e89b7-58ec-4857-b606-f3573bd4c501" />